packages/babel-preset-udemy-website/index.js

```diff
@@ -7,8 +7,9 @@ module.exports = {
         const name = it[0];
         const options = it[1] || {};
         // eslint-disable-next-line import/no-dynamic-require
-        const plugin = require(`babel-plugin-${name}`);
+        let plugin = require(`babel-plugin-${name}`);
         // https://github.com/loganfsmyth/babel-plugin-transform-builtin-extend/issues/1
-        return [name.includes('legacy') ? plugin.default : plugin, options];
+        plugin = plugin.default || plugin;
```

Comment:
The new plugin didn't work with the old code. This is a little more flexible.

Reply:
Nit you could keep plugin as const and do return [plugin.default || plugin, options]; I think it’d be a tiny bit less confusing.

Reply:
Done.